src/CommerceModel.BetterRetail/artifacts/OOE/BetterRetail/QueueOrderSchemaImportActivity/providers.json

```diff
@@ -497,11 +497,11 @@
       },
       "MonerisHostname": "esqa.moneris.com",
       "DefaultCulture": "en-CA",
-      "StoreId": "store1",
+      "StoreId": "store3",
```

Comment:
Because the QA needed to use store3 with the mentioned token

![image](https://user-images.githubusercontent.com/57448699/131685208-5abd4245-7a5b-4912-9206-dffb53645d93.png)

see comment from MQ https://dev.azure.com/orckestra001/OrckestraCommerce/_workitems/edit/50232

Reply:
Changed back to store1 with a new generated token
![image](https://user-images.githubusercontent.com/57448699/131723875-cc998817-83ab-4991-b5a8-6071fe573172.png)

ht4A7KKUOFOA7I9